_posts/2017-01-27-updates.markdown

```diff
@@ -0,0 +1,15 @@
+---
+layout: post
+title:  Mo Betta' - No Beta!
+date:   2017-01-27 08:00:00
+categories: updates
+---
+
+
+The world may seem a cold place right now, but warm your hearts with the news that our latest update, __v3.25.0__, is now available for download.
+
+Among the various updates included in this release, The API has been updated (to v1.13) to incorporate updates to `terms` filters to allow extension of MyReports models. Also, with the long-awaited completion of test suites and systems updates for Curriculum Inventory, we are able to remove the "Beta" tag from the CI, and continue into the coming year with full CI test coverage and code integrity.
```

Comment:
maybe say "Curriculum Inventory" once, spelled out fully, before using 'CI' to reference it